src/types.ts

```diff
@@ -1,5 +1,5 @@
 export interface StoreData {
```

Comment:
no need to use for an interface, you can use "type" keyword

```
export type Store Data {
    // ...
}
```

Reply:
[as the name says, type should be used for types i guess](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces)

no major differences